lib/js_locale_helper.rb

```diff
@@ -139,9 +139,9 @@ def self.output_MF(locale)
 
     message_formats =
       I18n.fallbacks[locale]
-        .each_with_object({}) do |l, hash|
+        .each_with_object(HashWithIndifferentAccess.new) do |l, hash|
```

Comment:
Using a hash with indifferent access, that way we don’t care if the locale is a symbol or a string.